recipes-kernel/linux/files/bcm2712-raspberrypi5-can.dtso

```diff
@@ -7,29 +7,30 @@
 /plugin/;
```

Comment:
commit title seems to be too long

Reply:
also you've mentioned that 

> Therefore, the device tree overlay bcm2712-raspberrypi5-can.dtso was
> adjusted to work properly with the MCP251XFD chip.
does it means that MCP2515 support was dropped? 
probably it worth adding both?